package.json

```diff
@@ -21,23 +21,20 @@
   },
   "devDependencies": {
     "@folio/eslint-config-stripes": "^3.2.1",
-    "@folio/stripes-connect": "^3.1.2",
-    "@folio/stripes-core": "^2.7.0",
+    "@folio/stripes": "^1.0.0",
     "babel-eslint": "^9.0.0",
     "eslint": "^5.6.0",
     "react": "^16.5.1",
-    "react-dom": "^16.5.1"
+    "react-dom": "^16.5.1",
+    "react-redux": "^5.0.7",
+    "redux": "^4.0.0"
   },
   "dependencies": {
-    "@folio/stripes-components": "^3.0.0",
-    "@folio/stripes-logger": "^0.0.2",
-    "@folio/vendors": "^1.0.0",
+    "@folio/vendors": "^1.1.0",
     "classnames": "^2.2.5",
     "prop-types": "^15.6.0"
   },
   "peerDependencies": {
```

Comment:
`"@folio/stripes": "^1.0.0"` is missing from peerDependencies

Reply:
Fixed.